ckanext/validation/tests/fixtures.py

```diff
@@ -19,3 +20,13 @@ def org():
 @pytest.fixture
 def dataset(org):
     return factories.Dataset(private=True, owner_org=org['id'])
+
+
+@pytest.fixture
+def mocked_responses():
+    from ckan.common import config
```

Comment:
This should use toolkit.

Reply:
\+

Reply:
This will be fixed and pushed by Alex shortly.